doc/lsst.obs.base/index.rst

```diff
@@ -38,11 +38,11 @@ Python API reference
 .. automodapi:: lsst.obs.base
    :no-main-docstr:
 
-.. automodapi:: lsst.obs.base.fitsExposureFormatter
+.. automodapi:: lsst.obs.base.formatters.fitsExposure
    :no-main-docstr:
 
-.. automodapi:: lsst.obs.base.fitsGenericFormatter
+.. automodapi:: lsst.obs.base.formatters.fitsGeneric
    :no-main-docstr:
 
-.. automodapi:: lsst.obs.base.fitsRawFormatterBase
+.. automodapi:: lsst.obs.base.formatters.fitsRawBase
```

Comment:
Did you miss this one?

Reply:
Yes. That moved back after our discussion and I think it can be deleted since it's a standard export anyhow.